build-logic-settings/cache-plugin/src/main/kotlin/convention-cache.settings.gradle.kts

```diff
@@ -12,9 +13,9 @@ val enterpriseUrl = stringProperty("avito.gradle.enterprise.url", nullIfBlank =
 
 buildCache {
     local {
-        isEnabled = booleanProperty("avito.gradle.buildCache.local.enabled", true)
+        isEnabled = booleanProperty("avito.gradle.buildCache.local.enabled", false)
         isPush = true
-        removeUnusedEntriesAfterDays = 30
+        removeUnusedEntriesAfterDays = 3
```

Comment:
3 seems like too low for this project, because it doesn't move so fast, and you will end up downloading much more from remote

Reply:
Sorry, it's my bad I forget to remove my local experiments